src/Bridge/Doctrine/Orm/ItemDataProvider.php

```diff
@@ -64,11 +65,18 @@ public function getItem(string $resourceClass, $id, string $operationName = null
         }
 
         $manager = $this->managerRegistry->getManagerForClass($resourceClass);
+        $classMetadata = $manager->getClassMetadata($resourceClass);
+        $metadataIdentifier = $classMetadata->getIdentifier();
```

Comment:
`$metadataIdentifier` is only used once (and the name is confusing). Let's make the call inline.